docs/maui/extensions/keyboard-extensions.md

```diff
@@ -0,0 +1,63 @@
+---
+title: KeyboardExtensions - .NET MAUI Community Toolkit
+author: PureWeen
+description: "The KeyboardExtensions provide a series of extension methods that support interacting with the Keyboard."
+ms.date: 04/12/2023
+---
+
+# KeyboardExtensions
+
+The `KeyboardExtensions` provide a series of extension methods that support interacting with the Keyboard..
```

Comment:
Should we mention that those methods can be used for `InputView` elements?

Reply:
updated
Let me know if that sounds alright?

Reply:
sounds amazing <3